src/cml.js

```diff
@@ -234,6 +236,89 @@ class CML {
     }
   }
 
+  async pr_create(opts = {}) {
+    const { globs = ['dvc.lock', '.gitignore'], md } = opts;
+
+    const { files } = await git.status();
+    if (!files.length) {
+      console.log('No files changed. Nothing to do.');
+      return;
+    }
+
+    const driver = get_driver(this);
+    const paths = (await globby(globs)).filter((path) =>
+      files.map((item) => item.path).includes(path)
+    );
+
+    const render_pr = (url) => {
+      if (md)
+        return `[CML's ${
+          this.driver === 'gitlab' ? 'Merge' : 'Pull'
+        } Request](${url})`;
+      return url;
+    };
+
+    const sha = await exec(`git rev-parse HEAD`);
+    const sha_short = sha.substr(0, 7);
+    let target = await exec(`git branch --show-current`);
+    if (!target) {
+      if (this.driver === 'gitlab') {
+        target = await exec('echo $CI_BUILD_REF_NAME');
+      }
+    }
+    const source = `${target}-cmlpr-${sha_short}`;
+
+    await exec(`git fetch origin`);
+
+    const branch_exists = (await exec(`git branch -r`)).includes(source);
+    if (branch_exists) {
+      const prs = await driver.prs();
+      const { url } =
+        prs.find((pr) => pr.source === source && pr.target === target) || {};
+
+      if (url) return render_pr(url);
+    } else {
+      try {
+        await exec(`git config --local user.email "david@iterative.ai"`);
```

Comment:
I assume that this is because we don't have a dedicated bot account yet. 🤔

Reply:
@0x2b3bfa0 yes. Thats not finished and actually the code is going to be reiterated. We are removing exec git in favour of an isomorphic git client. 
This release is to test it in your side, checking possible scenarios that I might not have thought about.

Reply:
Understood, thanks!